_episodes/09-pull-requests.md

```diff
@@ -84,32 +127,66 @@ $ git remote -v
 ~~~
 origin	https://github.com/USERNAME/countries.git (fetch)
 origin	https://github.com/USERNAME/countries.git (push)
-upstream	https://github.com/McMahonLab/countries.git (fetch)
-upstream	https://github.com/McMahonLab/countries.git (push)
+upstream	https://github.com/INSTRUCTOR-GIVEN/countries.git (fetch)
+upstream	https://github.com/INSTRUCTOR-GIVEN/countries.git (push)
 ~~~
 {: .output}
 
-Now that we have this setup done, which is only needed the first time, we can now suggest changes to this repo.
-Each person will add info about a specific country to the repo.
-Since you cloned the repo a template file with info on the United States has been added.
-First we will update our local version of the repo, with this change.
+Now that we have this setup done, which is only needed the first time, 
```

Comment:
Do you think it would be okay to get rid of that phrase all together? I was thinking 
"Now that we have this setup done we will be able to suggest changes to this repo using a pull request."